config/NGINX.md

```diff
@@ -0,0 +1,24 @@
+# If you want to use Nginx for a url without a port
```

Comment:
I'm sorry I remember why this works for me out of the box.

I have my machine setup with https://github.com/mikemcquaid/strap which sets this up for you.